python/version.py.in

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+
+# fmt: off
+#
+#   @CMAKE_GENERATED_COMMENT@
+#
+# fmt: on
+
+CCF_VERSION = "@CCF_VERSION@"
```

Comment:
If `CCF_VERSION` is long enough, then this (and the equivalent line in `version.h`) may end up too long and get reflowed. The formatting should be disabled for these lines as well.